l10n_it_ricevute_bancarie/models/account.py

```diff
@@ -46,8 +46,7 @@ def _check_protected_records(self):
         )
         if protected_records:
             message = _(
-                "The bank accounts with accreditation code "
-                f"{[bank.acc_number for bank in protected_records]}"
+                "One or more bank accounts with accreditation code "
                 " cannot be deleted as they are used in invoices."
                 " If possible, archive the bank account"
             )
```

Comment:
Grazie della PR.
Per evitare di perdere le informazioni potresti riformulare il codice in questo modo. Che ne pensi?

```         
            acc_numbers = [bank.acc_number for bank in protected_records]
            message = _(
                "The bank accounts with accreditation code {acc_codes}s"
                " cannot be deleted as they are used in invoices."
                " If possible, archive the bank account",
                acc_codes=', '.join(acc_numbers)
            )
```